scanapp/tasks.py

```diff
@@ -94,23 +150,23 @@ def save_results_to_db(scan_id, json_data, scan_type, URL, folder_name):
     insert_into_db = InsertIntoDB()
     scan_info = ScanInfo.objects.get(pk=scan_id)
     code_info = insert_into_db.insert_into_code_info(
-        scan_info = scan_info,
-        total_code_files = json_data['files_count'],
-        code_size = 4096,
+        scan_info=scan_info,
+        total_code_files=json_data['files_count'],
+        code_size=4096,
```

Comment:
How are these formatting changes related to your PR? Avoid mixing unrelated changes in a commit and PR, even for format related changes.